osbenchmark/workload/loader.py

```diff
@@ -1596,11 +1596,22 @@ def _create_test_procedures(self, workload_spec):
             schedule = []
 
             for op in self._r(test_procedure_spec, "schedule", error_ctx=name):
-                if "parallel" in op:
-                    task = self.parse_parallel(op["parallel"], ops, name)
+                if "clients_list" in op:
+                    self.logger.info("Clients list specified, running multiple search tasks with %s clients.", op["clients_list"])
```

Comment:
Nit: "Clients list specified, running task with multiple clients: %s" might be cleaner?

Reply:
I agree this is confusing. Technically we're running multiple tasks, each with multiple clients (or only 1 client).
I changed it to `self.logger.info("Clients list specified: %s. Running multiple search tasks, each scheduled with the corresponding number of clients from the list.", op["clients_list"])`